app/assets/stylesheets/views/_covid.scss

```diff
@@ -80,13 +72,27 @@
   }
 }
 
+.covid__logo-wrapper {
+  margin-bottom: govuk-spacing(7);
+
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: 0;
+    position: relative;
+    height: 108px;
+  }
+}
+
 .covid__logos {
   max-width: 100%;
   height: auto;
+  display: block;
+  margin: 0 auto;
 
   @include govuk-media-query($from: tablet) {
     margin-top: govuk-spacing(2);
```

Comment:
If this is absolutely positioned to the bottom, I think margin-top doesn't do anything and can be removed?